pkg/sensors/exec/cgroups_test.go

```diff
@@ -1068,3 +1141,415 @@ func TestCgroupv1K8sHierarchyInHybridBlkio(t *testing.T) {
 func TestCgroupv1K8sHierarchyInHybridInvalid(t *testing.T) {
 	testCgroupv1K8sHierarchyInHybrid(t, "invalid-cgroup-controller")
 }
+
+// This will execute a helper program that creates cgroups
+// then migrates to them and perform for each one exec in order
+// to assert collected cgroup id + name of the exec event match.
+// This works in Hybrid mode and is performed for all hierarchies.
+func testCgroupv1ExecK8sHierarchyInHybrid(t *testing.T, selectedController string) {
+	testutils.CaptureLog(t, logger.GetLogger().(*logrus.Logger))
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+
+	option.Config.HubbleLib = tus.Conf().TetragonLib
+	option.Config.Verbosity = 5
+
+	_, err := observer.GetDefaultObserver(t, ctx, tus.Conf().TetragonLib)
```

Comment:
Why is the observer used here? We shouldn't need it since we use the perf ring for testing.

Reply:
hmmm! we are loading tg_conf_map , doing execve checks here execve events...

Reply:
ok got it, will replace with tus.LoadSensor(ctx, t, base.GetInitialSensor()), thank you ;-)